### ISSUE TITLE
Create a model for security.txt

### ISSUE THREAD

**Issue by blink1073** [jupyter/security#3]
cf https://github.com/jupyter-server/jupyter_server/issues/249

We should have a standard method for handling [security.txt](https://en.wikipedia.org/wiki/Security.txt) files.  

Note that the one used by our main [website](https://github.com/jupyter/jupyter.github.io/blob/b954bd1f39b449991c6e4df559964019878c5e74/.well-known/security.txt) is from the notebook project.  Should the encryption file be generated per project?

**Comment by Carreau** [comment]
O don't think we need a per project encryption as vuln can anyway be across
projects.

**Comment by rpwagner** [comment]
I agree, we should limit the number of encryption keys, but have a simple policy on how to manage them. 

I submitted a [minimal SECURITY.md for JupyterHub](https://github.com/jupyterhub/jupyterhub/pull/3445). Something like this could be part of a repository template.
